examples/acoustic/acoustic_example.py

```diff
@@ -94,7 +94,7 @@ def run(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
     info("Applying Adjoint")
     Acoustic.Adjoint(rec, dse=dse, dle=dle, legacy=legacy)
     info("Applying Born")
-    Acoustic.Born(dm, dse=None, dle=dle, legacy=legacy)
+    Acoustic.Born(dm, dse=dse, dle=dle, legacy=legacy)
```

Comment:
Needs #233  to be merged first as this part is not tested with current setup. My fault, I removed the `full_run` t some point locally and forgot to put it back.